src/Compiler/CodeGen/EraseUnions.fs

```diff
@@ -871,12 +871,7 @@ let convAlternativeDef
         | SpecialFSharpListHelpers ->
 
             let baseTesterMeths, baseTesterProps =
-                if
-                    g.langVersion.SupportsFeature LanguageFeature.UnionIsPropertiesVisible
```

Comment:
Am wondering why this is removed

Reply:
This line was added recently for the auto-generated DU case tests (.Is*) visible from F# code.

However, the nullness feature (specifically the metadataexport) needs to generate those in a C# friendly way, so I reverted back to generating the IL for them in EraseUnions.

They are still visible from F# code, but the codegen for them happens in EraseUnions and not during typechecking.


(those export attributes are need to generate attributes which correlate case testers and data access. So that Roslyn can safely determin that e.g. `if( opt.IsSome) then opt.Value` is safe.